clink_dev/clink.iml

```diff
@@ -0,0 +1,371 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove the file for idea. And add the `.gitignore` file to prevent this file type from committing.